src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -599,7 +620,10 @@ private static bool IsCallingConventionModifier(CustomModifier modifier)
         {
             var modifierType = ((CSharpCustomModifier)modifier).ModifierSymbol;
             return (object)modifierType.ContainingAssembly == modifierType.ContainingAssembly.CorLibrary
-                   && modifierType.Name.StartsWith("CallConv", ignoreCase: false, culture: CultureInfo.InvariantCulture)
+                   && modifierType.Name != "CallConv"
+                   && modifierType.Arity == 0
```

Comment:
```suggestion
                   && modifierType.Arity == 0
                   && modifierType.Name != "CallConv"
```

Do the faster check first.